lib/builders/MaterialBuilder.ts

```diff
@@ -73,7 +73,12 @@ export class MaterialBuilder {
    * Outputs a static theme stylesheet, based on the specified options
    */
   _buildTheme(buildData: MaterialToolsData): MaterialToolsOutput {
-    if (!this._themeBuilder) {
+
+    // When the ThemeBuilder is not initialized and theme definitions are specified.
+    if (!this._themeBuilder && this._themes) {
+      let moduleName = path.join(buildData.package.module, this._options.mainFilename);

```

Comment:
We have this line repeated twice now ([also in `_getData`](https://github.com/angular/material-tools/blob/c6978d8d38a611aa9efe6a507ddb9af41d7cad8e/lib/builders/MaterialBuilder.ts#L40)). Couldn't we move the `path.join(buildData.package.module, this._options.mainFilename)` to[ the place where we update the version](https://github.com/angular/material-tools/blob/c6978d8d38a611aa9efe6a507ddb9af41d7cad8e/lib/builders/MaterialBuilder.ts#L35), after it was resolved?